prebid-mobile/pbm-api/android/pbm-plugin-renderer.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: page_v2
+title: Plugin Renderer - Android
+description: Guide to implement a plugin renderer
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+sidebarType: 2
+---
+
+# Prebid Plugin Renderer
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+Plugin Renderer is a feature that enable the ability to delegate the ad rendering to a component of yours. Such integration require from you, in first place, to have a Bidder Adapter implemented ([see more](prebid-server/developers/add-new-bidder-go.html)) in order to handle bid requests from the Prebid Mobile SDK that include your plugin renderer.
+
+Such feature turn possible, for instance, the rendering of non-standard ad responses that Prebid Mobile SDK can not render by itself.     
+
+![Plugin Renderer big picture](/assets/images/prebid-mobile/prebid-plugin-renderer.png)
+
+### Setup
+
+* Provide your Prebid Bidder Adapter ([see more](prebid-server/developers/add-new-bidder-go.html))
+* Create your implementation from the interface `PrebidMobilePluginRenderer`
+* Initialise your plugin renderer before starting to request ads
```

Comment:
```suggestion
* Initialise your plugin renderer before starting to request ads: this part has to be made by app owner (app integration)
```

Reply:
I'm not sure @elneruda, at least on Android a SDK could provide the activity where the plugin renderer would be initialised. Then the app owner does nothing at all

Reply:
We can clarify that such implementations should be done externally not in the PBM SDK itself

Reply:
I updated the PR with a clarification just below the setup list